deadlines/src/main/java/com/palantir/deadlines/Deadlines.java

```diff
@@ -112,30 +112,23 @@ public static void disableFurtherDeadlinePropagation() {
      * this value and one already set via a previous call to {@link #parseFromRequest}, if it exists
      * @param request the request object to write the encoding to
      * @param adapter a {@link RequestEncodingAdapter} that handles writing the header value to the request object
+     * @param enforcement a client requested {@link Enforcement} state, which will be resolved against any existing state
```

Comment:
> which will be resolved against any existing state

we should clarify what this means in the javadoc here, given that `resolveEnforcementStrategy` is private to this class.